tests/integ/scala/test_dataframe_suite.py

```diff
@@ -980,6 +981,86 @@ def test_select(session):
     # select(Seq[Column]) with col("a") + col("b")
     assert df.select([col("b"), col("a") + col("c")]).collect() == expected_result
 
+    # test single output column udtf
+    class TwoXUDTF:
+        def process(self, n: int) -> int:
+            yield (2 * n,)
+
+    table_func = udtf(
```

Comment:
Should we also test snowflake-built-in table function here? (As what you have in doctest)

Reply:
good suggestion. Added a similar test